src/test/java/edu/iis/powp/gui/TestPlotterApp.java

```diff
@@ -65,11 +66,13 @@ private static void setupDrivers(Application application) {
 		application.addDriver("Client Plotter", clientPlotter);
 
 		DrawPanelController drawerController = DrawerFeature.getDrawerController();
-		IPlotter plotter = new LineAdapterPlotterDriver(drawerController, LineFactory.getBasicLine(), "basic");
+		LineAdapterPlotterDriver linePlotter = new LineAdapterPlotterDriver(drawerController, LineFactory.getBasicLine(), "basic");
+		IPlotter plotter = new LinePlotterInkControlDriver(linePlotter);
```

Comment:
FIX TO MERGE: Still, this functionality is not optional (hardwired). If a new driver will be added it cannot be automatically equipped with InkController. It should be possible to attach LinePlotterInkControlDriver using observer pattern implemented in DriverManager. There should be also an option to TURN OFF ink control (it is hard to use this feature at all). When the application starts the decorator could be attached in a separate setupInkController method.